Assets/Scripts/Inventory/Chest.cs

```diff
@@ -0,0 +1,170 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Chest : PlacedObject
+{
+    public List<Item> contents = new List<Item>();
+
+    public GlobalStorage globalStorage;
+
+    public Vector3 coordinate;
+
+    /// Basic constructor for chest - sets it up so that it hooks up w/ GlobalStorage system
+    public Chest(GlobalStorage gs)
```

Comment:
It's a bit of a cop out and I'd/I'll have to look into it, but an alternative to requiring a reference to GlobalStorage passed into the constructor for each new chest generation is to have add a reference to GlobalStorage in the "GlobalInstance" singleton which could be accessed then whenever/wherever. I do know Singletons are largely disliked and tend to be abused, from what I've read, so this approach is definitely fine for now.